packages/office-ui-fabric-react/src/components/ColorPicker/ColorRectangle/ColorRectangle.base.tsx

```diff
@@ -81,16 +102,68 @@ export class ColorRectangleBase extends React.Component<IColorRectangleProps, IC
     );
   }
 
-  private _onMouseDown = (ev: React.MouseEvent<HTMLElement>): void => {
+  private _onKeyDown = (ev: React.KeyboardEvent<HTMLElement>): void => {
+    const { color } = this.state;
+    let { s, v } = color;
+
+    const increment = ev.shiftKey ? 10 : 1;
+
+    // Intentionally DO NOT flip the color picker in RTL: its orientation is not very meaningful,
+    // and getting all the math and styles flipped correctly is tricky
+    switch (ev.which) {
```

Comment:
hmmm.

Reply:
I'm open to trying to do RTL, it's just A LOT of things to get right (some of which were totally wrong before this PR). Copying from the description:

- Static gradient styles handled by merge-styles (easy)
- Gradient styles not handled by merge-styles (due to frequent updates)
- Mouse movement in slider
- Mouse movement in rectangle
- Keyboard in slider
- Keyboard in rectangle
- Thumb positioning in slider (not handled by merge-styles due to frequent updates)
- Thumb positioning in rectangle (same)

Reply:
Maybe ok to save for another day.

Reply:
Opened #10900 to track